### PR TITLE
Implementation of modulo line numbers

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`take row 15` => will select 15/115/215/...

### CODE REVIEW

File: src/processTargets/processMark.ts [hunk 1]
```diff
@@ -151,10 +151,27 @@ function processDecoratedSymbol(
...
+  const editor = context.currentEditor!;
+  const getLine = (linePosition: LineNumberPosition) => {
+    switch (linePosition.type) {
+        case "absolute":
+            return linePosition.lineNumber;
+        case "relative":
+            return editor.selection.active.line + linePosition.lineNumber;
+        case "modulo100":
+            const stepSize = 100;
+            const { start, end } = editor.visibleRanges[0];
```
[the remark this answers is not in the document]
Comment (AndreasArvidsson) [reply src/processTargets/processMark.ts]: Good call. I will just throw an error if I can find multiple matching lines.

[the remark this answers is not in the document]
Comment (AndreasArvidsson) [reply src/processTargets/processMark.ts]: @pokey It's updated and looks good imnsho :)

**Comment by AndreasArvidsson** [comment]
@pokey So now we have proper support for folded regions

**Review by pokey — APPROVED** [review]
Looks good.  I ran the yaml normaliser in https://github.com/pokey/cursorless-vscode/pull/333/commits/4e1c57a116d0076a4f8793e3b7a0580d056415de to make future diffs cleaner.  We should maybe bake this step into pre-commit / CI

**Pull request merged**